src/View/Helper/TimeHelper.php

```diff
@@ -86,35 +86,35 @@ public function nice($dateString = null, $timezone = null, $locale = null)
     }
 
     /**
-     * Returns true if given datetime string is today.
+     * Returns true, if the given datetime string is today.
```

Comment:
It seems it is _unnecessary_, but __not prohibited__.

> Rule 4b. A comma is usually unnecessary when the sentence starts with an independent clause followed by a dependent clause.

But even the Oxford dictionary seem to use them in some cases:

> You also need to use a comma at the end of a piece of direct speech, if the speech comes before the information about who is speaking.

https://en.oxforddictionaries.com/punctuation/comma#commas_in_direct_speech

Anyway, my intend with the comma was to make the statement more clear...